desktop/package.json

```diff
@@ -86,5 +88,8 @@
     "webpack-dev-server": "1.14.1",
     "webpack-hot-middleware": "2.10.0",
     "webpack-target-electron-renderer": "0.4.0"
+  },
+  "optionalDependencies": {
+    "fsevents": "1.0.11"
```

Comment:
`fsevents` fails to install on Linux because it is only available on OSX. Adding it to `optionalDependencies` allows the install to continue in this case (even when shrinkwrapped!). I tried to document this in the commit message, since we can't put commens in the JSON: https://github.com/keybase/client/pull/2822/commits/c8478e3d9fe1261fb54df40140015fdf8052eeef